.vscode/launch.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
don't commit the temporary workspace files

Reply:
already remove